lib/mega.js

```diff
@@ -1,7 +1,9 @@
 import { megaEncrypt, megaDecrypt, megaVerify } from './crypto/index.mjs'
 import Storage from './storage.mjs'
+import MutableFile from './mutable-file.mjs'
 import File from './file.mjs'
 import API from './api.mjs'
+import MutableFile from './mutable-file.mjs'
```

Comment:
Imported it twice?

Reply:
Oh my goodness, how could I write it twice

Reply:
It happens sometimes, people are people and people make mistakes, that's why there are code reviews 😄